notes.md

```diff
@@ -0,0 +1,124 @@
+## other notes.
+
+### Terms
+
+  - Price per share approx. `$54`
+  - Nature of securities `Equity`
+  - Closing Date `May 30 (Ideally)`
+  - Amount of securities offered `Approx 55,000`
+
+### Implied Terms
+  - Statement as to the number of days left in the campaign `probably 30 but TBD`
+  - Offering goal `$2.5M`
+
+### Non-terms
+
+* didn't come across as super relevant right now. Closing soon can appear on May 23 perhaps, one week before close.
+
+### Releasing of funds and delayed founder liquidity-
+
+  - both of these concerns are alleviated by the corporate structure.
+
+      1. Invested funds must be used for intended purposes or shareholders can sue us, thereby alleviating the risk of immediately releasing funds.
+      2. We have delayed liquidity in the sense that the company won't own a `premine`, so if we don't continue to perform valuable work for the network, people won't agree to pay our fee on block rewards and fork.
+
+  - Authorized, but unissued shares can be held in an address owned by the company.
+  - We do have to differentiate between never issued and repurchased tokens (treasury shares), however.
```

Comment:
If we had a counter for the number of times a token has been reissued it could be a good way to differentiate.

Reply:
what does it mean for a token to be re-issued?  Tokens can either be 'minted' (first created) or 'Transferred' and that's about it.

Reply:
@tavakyan separate company-owned addresses for treasury vs unissued stock is probably the best way to manage it. Unissued stock address can send but not receive.